caffe2/operators/lengths_reducer_fused_8bit_rowwise_ops.cc

```diff
@@ -28,7 +28,7 @@ OPERATOR_SCHEMA(SparseLengthsSumFused8BitRowwise)
     .SetDoc(R"DOC(
 Performs the same operation as SparseLengthsSum, but operating on
 8-bit rowwise quantized matrices with fused storage (where each row
-stores the scale, bias and then quantized values).
+stores quantized values, and then the scale and bias).
```

Comment:
Is it worth going into detail, e.g. "and then 8 bytes: 4 bytes for scale and 4 bytes for bias" or is that obvious given the context? (same tweak could apply for the other comment corrections below)

Reply:
Good point. Will do.